MeshLib/MeshEditing/Mesh2MeshPropertyInterpolation.h

```diff
@@ -31,32 +31,28 @@ class Mesh2MeshPropertyInterpolation final
      * Constructor taking the source or input mesh and properties.
      * @param source_mesh the mesh the given property information is
      * assigned to.
-     * @param source_properties a vector containing property information
-     * assigned to the mesh. The number of entries in the property vector
-     * must be at least the number of different properties stored in mesh.
-     * For instance if mesh has \f$n\f$ (pairwise) different property
-     * indices the vector of properties must have \f$\ge n\f$ entries.
+     * @param property_name is the name of a PropertyVector in the \c
+     * source_mesh
      */
-    Mesh2MeshPropertyInterpolation(Mesh const*const source_mesh, std::vector<double> const*const source_properties);
+    Mesh2MeshPropertyInterpolation(Mesh const* const source_mesh,
```

Comment:
A `nullptr` should not be a valid input. You are right, I'll change this to an refernce.